build.sh

```diff
@@ -1 +1,12 @@
-cmake --build build/linux --config profile --target ROS2-Gem-Demo Editor AssetProcessor -j4
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# helper script to rebuild project
+
+cmake --build build/linux --config profile --target ROS2-Gem-Demo Editor AssetProcessor -j12
```

Comment:
We will want to rename ROS2-Gem-Demo to RobotVacuumSample, but that can be done in a different PR